src/basics.md

```diff
@@ -847,6 +848,31 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-check-cpu-cores]: #ex-check-cpu-cores
+<a name="ex-check-cpu-cores"></a>
+## Check number of logical cpu cores
+
+[![num_cpus-badge]][num_cpus] [![cat-hardware-support-badge]][cat-hardware-support]
+
+Counts the number of logical cpu cores in current machine.
```

Comment:
I would rephrase that with something along the lines of:
"Shows the number of logical cpu cores with help of `num_cpus::get`" and I would hyperlink the symbol reference to its respective docs.rs

Reply:
I hope the heading is fine as it is.